src/tests/integration/konvert/test_simple_proofs.py

```diff
@@ -362,3 +378,33 @@ def test_kore_to_kast(
 
         # Then
         assert actual_kast == kast
+
+    @pytest.mark.parametrize(
+        'rule_id,kore_text',
+        KRULE_TO_KORE_DATA,
+        ids=[rule_id for rule_id, *_ in KRULE_TO_KORE_DATA],
+    )
+    def test_krule_to_kore(
+        self,
+        kprint: KPrint,
+        rule_id: str,
+        kore_text: str,
+    ) -> None:
+        # cannot do the following because mypy complains:
+        # `error: "KDefinition" has no attribute "main_module"; maybe "main_module_name"?``
+        # main_module = kprint.definition.main_module
+        k_definition = kprint.definition
+        ms = [m for m in k_definition.modules if m.name == k_definition.main_module_name]
+        assert len(ms) == 1
+        main_module = ms[0]
```

Comment:
Look through the methods of `class KDefinition`, you have `KDefinition.all_modules_dict`.

```suggestion
        main_module = kprint.definition.all_modules_dict[kprint.definition.main_module_name]
```

Reply:
We should consider adding `@property main_module` that is just this implementation, as @tothtamas28 suggested.